src/math.sol

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2018 Rain <rainbreak@riseup.net>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+contract Math {
+    uint256 constant ONE = 10 ** 27;
+
+    function rpow(uint x, uint n, uint base) public pure returns (uint z) {
```

Comment:
I would move the rpow function into interest.sol given most of the time we don't ever use it for anything else.

Reply:
I think it makes more sense to keep it in math, as it's a generic safe-math function